recipes/bert-tensorflow/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "bert-tensorflow" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 979ab38715be88bc95483654994c8bbb85acdbfdc60ca1a0ff90cfe8c8f95ea8
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - six
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - bert
+
+about:
+  home: "https://github.com/google-research/bert"
+  license: Apache 2.0
+  license_family: APACHE
```

The SPDX identifier has a dash, you can then remove the `license_family` field:
```suggestion
  license: Apache-2.0
```